### DISCUSSION TITLE
Unable to resolve custom scheme on nuxt start?

### DISCUSSION THREAD

**Discussion by kan01234** [nuxt-community/auth-module#1463] [Q&A]
i am following this guide to add custom scheme
https://auth.nuxtjs.org/guide/scheme

nuxt.config.js
```
  auth: {
    strategies: {
      customStrategy: {
        scheme: '~/schemes/customScheme',
...
```

but when I run nuxt build and start, it return error
```
Error: Cannot resolve "~/schemes/customScheme" from "/src/pj/schemes/customScheme"
```

**Comment by Ahmard** [comment]
Is there any fix around this?

**Comment by TonalidadeHidrica** [comment]
I'm also having the exactly same problem.  Could somebody help us out?

**Comment by cbyte** [comment]
@TonalidadeHidrica in the docs the custom scheme is shown as a javascript file. Try to refer it in your nuxt.config.js like that:
```
scheme: '~/schemes/customScheme.js',
```

**Reply by TonalidadeHidrica** [reply]
@cbyte Thanks for your blazing fast reply!  Actually, I was using `.ts` custom scheme source.  Indeed, specifying `scheme: '~/schemes/customScheme.ts',`instead of `scheme: '~/schemes/customScheme',` blew out the problem!!  Thank you so much, you saved my day.

P.S. If the scheme code was ordinary JavaScript file i.e. `~/schemes/customScheme.js`, specifying `scheme: '~/schemes/customScheme',` was enough.  I verified it by generating minimum nuxt project.  This project seems occurring only the file was TypeScript.

**Comment by AndreyMashukov** [comment]
Today I had the same problem, and the problem was: when I build my app in Dockerfile, I forgot to copy folder with schema, so, if you use docker make sure that you have this file in the container
